### DISCUSSION TITLE
Original Tonie Figure

### DISCUSSION THREAD

**Discussion by scratchplate** [MiczFlor/RPi-Jukebox-RFID#1482] [Q&A]
Hello,

i´ve a short question about the Phonie project.

Can someone tell me, if the Phonie Box works with the Original figures?
I´ve already bought the EM4100 Reader. Can the EM4100 read the original Tonie figures, that i configure it as a normal 125khz chip?

Greetings

**Comment by s-martin** [comment]
See #992.
It makes sense to continue the discussion there
